### DISCUSSION TITLE
Bookie handle is not available while reading L309 E2 from bookie: 127.0.0.1:3181

### DISCUSSION THREAD

**Discussion by AaqibHasaniee** [apache/pulsar#23490] [Q&A]
I am trying to run a standalone pulsar cluster in docker. I was running the following command, which was able to setup a local cluster:

```
docker run -it -p 6650:6650  -p 8080:8080 --mount source=pulsardata,target=/pulsar/data --mount source=pulsarconf,target=/pulsar/conf apachepulsar/pulsar:2.10.6 bin/pulsar standalone
```

However, this command has changed over the weekend to:
```
docker run -it \
-p 6650:6650 \
-p 8080:8080 \
--mount source=pulsardata,target=/pulsar/data \
--mount source=pulsarconf,target=/pulsar/conf \
apachepulsar/pulsar:3.3.2 \
bin/pulsar standalone
```

Now, whenever i run the above command in my local vs code, i face the error:

```
Bookie handle is not available while reading L309 E2 from bookie: 127.0.0.1:3181 and this errror keeps on appearing forever. 
```

I have tried doing PULSAR_STANDALONE_USE_ZOOKEEPER=1 in the env file, however it does not work.

**Comment by yuweisung** [comment]
2.10.6 to 3.3.2 is a big jump and there are some additional topics/metadata may miss during the upgrade. 
you can use bookie cli to trace which ledger is missing and what topic owns the ledger.
